executor/linux/step.go

```diff
@@ -388,3 +388,21 @@ func (c *client) loadStepLogs(name string) (*library.Log, error) {
 
 	return l, nil
 }
+
+// loadStep is a helper function to capture
+// a step from the client.
```

Comment:
I think this comment needs to be updated 👍

```suggestion
// loadInitContainer is a helper function to capture
// the init step from the client.
```